pandas/core/arrays/integer.py

```diff
@@ -748,7 +743,7 @@ def integer_arithmetic_method(self, other):
 
             # nans propagate
             if mask is None:
-                mask = self._mask
+                mask = self._mask.copy()
```

Comment:
With just this change, the truediv tests failed on master.  We apparently didn't catch it because the masked values weren't being compared in `_checkop`